### PR TITLE
Port .travis.yml to github actions

### PR DESCRIPTION
**Pull request by gherceg**
Run tests using github actions instead of travis.

I didn't port over this line from the .travis.yml:
```
jobs:
  allow_failures: 
  - python: "3.7"  # because pip install coveralls fails
  ```
  since the tests appeared to run just fine as is. I also plan to remove 3.7 from the matrix since it has been EOLed.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        python-version: [ "3.7", "3.8", "3.9" ]
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: install dependencies
+        run: |
+          pip install django==3.2.*
```
Comment (millerdev) [inline .github/workflows/tests.yml:24]: Might be worth adding another matrix dimension for Django versions and run against 3.2 and 4.2.

**Review by millerdev — APPROVED** [review]
Fine to address the comments in a separate PR since this appears to provide near feature parity with what it's replacing.

**Comment by gherceg** [comment]
> Fine to address the comments in a separate PR since this appears to provide near feature parity with what it's replacing.

Was planning on doing just that! Thanks 👍🏻

File: .github/workflows/tests.yml (hunk 1 above)
Comment (gherceg) [reply .github/workflows/tests.yml:24]: See https://github.com/dimagi/django-prbac/pull/72

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    
+    strategy:
+      matrix:
+        python-version: [ "3.7", "3.8", "3.9" ]
```
[the remark this answers is not in the document]
Comment (gherceg) [reply .github/workflows/tests.yml:15]: See https://github.com/dimagi/django-prbac/pull/72

**Comment by gherceg** [comment]
Addressed comments in https://github.com/dimagi/django-prbac/pull/72

**Pull request merged**